.github/workflows/CI.yml

```
@@ -14,36 +14,41 @@ jobs:
-        echo -n "$CREATE_SECRET" | base64 --decode > $CREATE_SECRET_DIR/$CREATE_SECRET_DIR_FILE_NAME
-
-    - name: 5. Grant execute permission for gradlew
-      run: chmod +x gradlew
-
-    - name: 6. Build with Gradle without test
-      run: ./gradlew clean build -x test
+      - name: 1. checkout
+        uses: actions/checkout@v4
+
+      - name: 2. Set up JDK 17
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)